src/swarm/neo/client/RequestSet.d

```diff
@@ -349,18 +351,15 @@ public final class RequestSet: IRequestSet
         ***********************************************************************/
 
         public void startOnNewConnection ( )
-        in
         {
-            assert(this.id);
-            assert(this.type == Type.multi_node);
+            verify(this.type == Type.multi_node);
 
-            auto rq = this.id in this.outer.active_requests;
+            assert(this.id);
```

Comment:
I feel it'd be a good idea to add a small comment stating where `assert`s have already been validated with `verify` or `enforce`. Otherwise, there's always the question "was this missed?".

Reply:
Actually, this is that kind of `assert` that is needed only during early debugging stages, it is much like an `invariant`. But you're right, it deserves a comment.